slides.md

```diff
@@ -468,7 +468,10 @@ const loaders = {
   Pet: {
     async owner(queries, context) {
       const petNames = queries.map(({ obj }) => obj.name)
-      return ownersByPetNames(context.app.pg, petNames)
+      const owners = await ownersByPetNames(context.app.pg, petNames)
+      return queries.map(({ obj: pet }) =>
+        owners.find(owner => owner.id === pet.owner)
+      )
```

Comment:
if we order in sql, do we still need to order in code?

Reply:
Sorry, noticed just after pushing. Fixed.